types/components/Alert.d.ts

```diff
@@ -0,0 +1,12 @@
+import { BoolPropsFromArray, GeneratedPropTypes } from '../types';
+import { stateTypes } from '../helpers/applyStateColor';
+import React from 'react';
+export declare type AlertProps = {
+    button?: any;
+    icon: string;
+    title?: string | React.ReactNode;
+    message?: string | React.ReactNode;
+    handleClose?: any;
```

Comment:
can this be something like
```suggestion
    handleClose?: () => void;
```
?